### ISSUE TITLE
As a project coordinator, I want the task list statuses to be clear, so that I understand the state of my work

### ISSUE THREAD

**Issue by mikevespi** [bcgov/cas-cif#540]
#### Description:

Based on user testing, the current statuses in the task list are misleading. Replace them with:
- Filled	
- No Changes
- Attention Required
- In Progress (when that one form is being edited and on screen)
- Not Started (?) for forms that have not yet been filled in when creating a new project.

#### Acceptance Criteria:

Given I am a Project Coordinator,
When I am creating a new project,
Then forms I have not yet filled in should show "Not Started", forms not submitted but with fields filled should show "In Progress", submitted forms should show "Filled" and forms with errors should show "Attention Required".

Given I am a Project Coordinator,
When I am editing an existing project (creating a new project revision),
Then forms I have not yet filled in should show "No Changes", forms not submitted but with changed fields should show "In Progress", forms with submitted changes should show "Filled" and forms with errors should show "Attention Required".

Given I am a Project Coordinator,
When I am viewing a committed project revisions forms,
Then no statuses should be shown.

#### Development Checklist:

- [ ] Task list shows correct statuses when creating a project
- [x] Task list shows correct statuses when updating a project revision
- [x] Meets the DOD

**Comment by mikevespi** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/540?planning-poker) with ZenHub @dleard @matthieu-foucault @pbastia @gurjmatharu @BCerki @Sepehr-Sobhani

**Comment by Sepehr-Sobhani** [comment]
The first paragraph of this issue can be covered partially at this stage since it's linked to a bigger issue (Bug #612). We need to change the way we calculate the status for each task list form and then we can completely cover the acceptance criteria.

**Comment by LindsayMacfarlane** [comment]
Thanks!

Observation: 
- Project contacts started off as 'in progress', while project managers and quarterly reports were not started.
- I don't understand 'filled'. Is that supposed to be completed? I'm not the user, but that doesn't seem very intuitive. 
- I couldn't trigger the attention required status - any ideas?

**Comment by matthieu-foucault** [comment]
The use of the word "filled" was chosen to replace "completed" to prevent confusion when, according to the current designs the task list also had information relative to the status of the reporting: "completed" could be misunderstood as "the milestone was completed". The redesign in progress by @suhafa may remove that confusion, and, if "filled" doesn't make sense to you, we could revert this change

**Comment by matthieu-foucault** [comment]
- The issue with contacts is known and being fixed in another issue. 
- To trigger the attention required status, creating quarterly reports without selecting any due date and submitting that form should trigger the change

**Comment by LindsayMacfarlane** [comment]
Thank you for the clarification @matthieu-foucault. I re-tested and was able to trigger the 'attention required'. We can wait on updating the 'filled' until we have Suha's design's and validation from the users.
